### PR TITLE
923 Let's implement RtRepo.languages().

### PR DESCRIPTION
**Pull request by nhekfqn**

### CODE REVIEW

**Comment by nhekfqn** [comment]
@dmarkov @krzyk this pull request is a replacement of https://github.com/jcabi/jcabi-github/pull/961 all original commits have been squashed into one and rebased. Please review.

Sorry for making separate PR, should have made it from separate branch in first place but didn't.

**Comment by dmarkov** [comment]
Thanks for the pull request, let me find a reviewer..

**Comment by dmarkov** [comment]
@krzyk it's yours, please review

**Comment by krzyk** [comment]
@rultor merge pls

File: src/main/java/com/jcabi/github/RtRepo.java [hunk 1]
```diff
@@ -217,8 +218,31 @@ public Notifications notifications() {
...
-        return Collections.emptyList();
+        final RtJson rtJson = new RtJson(
+            this.request.uri()
+                .path("/languages")
+                .back()
+        );
+        final JsonObject json;
+        try {
+            json = rtJson.fetch();
+        } catch (final IOException ex) {
```
Comment (yegor256) [inline src/main/java/com/jcabi/github/RtRepo.java]: I think it's better to let this exception bubble up and declare it in `throws` of the method. this is how all other methods in the library work

File: src/main/java/com/jcabi/github/RtRepo.java [hunk 2]
```diff
@@ -217,8 +218,31 @@ public Notifications notifications() {
     }
 
     @Override
+    @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
     public Iterable<Language> languages() {
-        return Collections.emptyList();
+        final RtJson rtJson = new RtJson(
```
Comment (yegor256) [inline src/main/java/com/jcabi/github/RtRepo.java]: I would name this `json` and the other one `object`, see http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

**Comment by yegor256** [comment]
@nhekfqn a few comments from me

**Comment by nhekfqn** [comment]
@yegor256 Please review.

File: src/main/java/com/jcabi/github/RtRepo.java [hunk 3]
```diff
@@ -217,8 +218,27 @@ public Notifications notifications() {
...
-        return Collections.emptyList();
+    @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
+    public Iterable<Language> languages() throws IOException {
+        final RtJson json = new RtJson(
+            this.request.uri()
+                .path("/languages")
+                .back()
+        );
+        final JsonObject object;
+        object = json.fetch();
```
Comment (yegor256) [inline src/main/java/com/jcabi/github/RtRepo.java]: I think these two lines can be joined :)

**Comment by yegor256** [comment]
@nhekfqn one more comment above

**Comment by nhekfqn** [comment]
@yegor256 Fixed. Please review.

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by dmarkov** [comment]
@krzyk Thanks for your contribution, **17 mins** was added to your account, payment ID is `50300846`; +17 to your rating, your total score is [+1680](http://www.netbout.com/b/35035?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy

**Pull request merged**